deploy/iso/minikube-iso/package/crio-bin/crio-bin.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/Makefile b/Makefile
+index de79f63..a9ea13b 100644
+--- a/Makefile
++++ b/Makefile
+@@ -136,7 +136,7 @@ crio.conf: bin/crio
+ release-note: ${RELEASE_TOOL}
+ 	${RELEASE_TOOL} -n $(release)
+ 
+-conmon/config.h: git-vars cmd/crio-config/config.go oci/oci.go
++conmon/config.h: cmd/crio-config/config.go oci/oci.go
```

Comment:
So the git executable is not available within your build environment? Because the git-vars target was introduced to actually fix this, but it only checks for the `.git` folder. I think I can improve this :)

Reply:
Actually it was worse... We do have git, but that's for our buildroot. Thus it recorded [commits from a different project](https://github.com/kubernetes/minikube/issues/3454). We build all our packages from tarballs, so have to supply the matching commit for each one.

The main problem here is that the "git-vars" target forces the conmon/config.h to be re-generated, even if it already exists (same with crio.conf too)

Reply:
Ah I see, thanks. Let's see if we can get rid of the git-vars dependency of this target entirely.